spec/services/attribute_asserter_spec.rb

```diff
@@ -316,11 +367,24 @@
         end
 
         context 'authn request specifies bundle with first_name, last_name, email, ssn, phone' do
+          # rubocop:disable Layout/LineLength
           let(:raw_sp1_authn_request) do
+            request_url = saml_authn_request_url(
+              saml_overrides: {
+                issuer: 'https://rp1.serviceprovider.com/auth/saml/metadata',
+                authn_context: [
```

Comment:
I see a handful of these `authn_context` arrays. Would it have been helpful to keep around a helper to generate them?

Reply:
Maybe, but they're all a bit different, so I'll leave that to a future PR.